docs/troubleshooting.md

```diff
@@ -3,51 +3,50 @@
-[here](https://grafana.com/grafana/dashboards/11575).
-
+  `/metrics`. A grafana dashboard can be found
+  [here](https://grafana.com/grafana/dashboards/11575).
 - Logs can be helpful in identifying issues. The verbosity level, which
-defaults to `INFO` can also be adjusted by passing the `--log-level` flag
-to the `otelcol` process. See `--help` for more details.
-
-- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#local-exporters)
-can be configured to inspect the data being processed by the Collector.
-
+  defaults to `INFO` can also be adjusted by passing the `--log-level` flag
+  to the `otelcol` process. See `--help` for more details.
+- [Local exporters](https://github.com/open-telemetry/opentelemetry-collector/tree/master/exporter#general-information)
+  can be configured to inspect the data being processed by the Collector.
 - The [health_check](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/healthcheckextension/README.md)
-extension, which by default is on port `13133`, can be used to ensure
-the Collector is functioning properly.
-
-- The [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/zpagesextension/README.md)
-extension, which by default is on port `55679`, can be used to check
-receivers and exporters trace operations via `/debug/tracez`.
-
+  extension, which by default is on port `13133`, can be used to ensure
+  the Collector is functioning properly.
 - The [pprof](https://github.com/open-telemetry/opentelemetry-collector/tree/master/extension/pprofextension/README.md)
-extension, which by default is on port `1777`, allows you to profile the
-Collector as it runs.
+  extension, which by default is on port `1777`, allows you to profile the
+  Collector as it runs.
 
 ## Common Issues
 
 ### Collector exit/restart
 
-The Collector may exit/restart if it is not
-[sized properly](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/performance.md)
-or configured properly, for example a queue size configured higher than
-available memory. In most cases, restarts are due to memory
-pressure. To mitigate this, we recommend you configure the
-[memory_limiter](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/memorylimiter/README.md)
-processor.
+The Collector may exit/restart because:
 
-Note: restarts may be due to resource limits configured on the Collector.
+- Memory pressure due to missing or misconfigured
+  [memory_limiter](https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/memorylimiter/README.md)
+  processor.
+- [Improperly sized](https://github.com/open-telemetry/opentelemetry-collector/blob/master/docs/performance.md)
+  for load.
+- Improperly configured (for example, a queue size configured higher
+  than available memory).
+- Infrastructure resource limits (for example Kubernetes).
```

Comment:
I'm not sure about this point. Aren't all these reasons essentially because of resource limits?